src/Ombi/ClientApp/src/app/components/detailed-card/detailed-card.component.scss

```
@@ -1,63 +1,215 @@
+    inset: 0;
     background-size: cover;
-    background-repeat: no-repeat;
-    background-position: center center;
+    background-position: center;
+    opacity: 0;
+}
 
-    @media (min-width: 768px) {
-        width: 400px;
-        height: 250px;
+::ng-deep .card-poster {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)